app/Http/Controllers/Dashboard/DashboardController.php

```diff
@@ -79,9 +79,26 @@ public function showDashboard()
         $components = Component::orderBy('order')->get();
         $incidents = $this->getIncidents();
         $subscribers = $this->getSubscribers();
-        $usedComponentGroups = Component::enabled()->where('group_id', '>', 0)->groupBy('group_id')->pluck('group_id');
-        $componentGroups = ComponentGroup::whereIn('id', $usedComponentGroups)->orderBy('order')->get();
-        $ungroupedComponents = Component::enabled()->where('group_id', 0)->orderBy('order')->orderBy('created_at')->get();
+
+        // Component & Component Group lists.
+        $usedComponentGroups = Component::enabled()->where('group_id', '>', 0)
+            ->groupBy('group_id')
+            ->pluck('group_id')
+        ;
+        $componentGroupsBuilder = ComponentGroup::guest();
+        if (auth()->check()) {
+            $componentGroupsBuilder = ComponentGroup::loggedIn(auth()->user());
+        }
+        $componentGroups = $componentGroupsBuilder->whereIn('id', $usedComponentGroups)
+            ->orderBy('order')
+            ->get()
+        ;
+        $ungroupedComponents = Component::enabled()
+            ->where('group_id', 0)
+            ->orderBy('order')
+            ->orderBy('created_at')
+            ->get()
+        ;
```

Comment:
You don't need to run PHP-CS-Fixer :) just write your code and StyleCI will fix it for you.